ui/raidboss/data/05-shb/raid/e12n.txt

```diff
@@ -6,5 +6,154 @@ hideall "--sync--"
+# -ic "Lissom Sculpture" "Beastly Sculpture"
+
+
 0.0 "--sync--" sync /:Engage!/ window 0,1
+1.0 "--sync--" sync /:Eden's Promise:4B1D:/ window 1,1
+14.9 "Maleficium" sync /:Eden's Promise:5872:/ window 14.9,5
+22.1 "--sync--" sync /:Eden's Promise:5879:/
+25.3 "Initialize Recall" sync /:Eden's Promise:5859:/
+34.9 "Judgment Jolt" sync /:Eden's Promise:585F:/
+45.6 "Temporary Current" sync /:Eden's Promise:585C:/
+56.3 "Judgment Jolt" sync /:Eden's Promise:585F:/
+67.0 "Temporary Current" sync /:Eden's Promise:585C:/
+80.0 "Formless Judgment" sync /:Eden's Promise:5873:/
+87.3 "--sync--" sync /:Eden's Promise:5879:/
+96.7 "Conflag Strike" sync /:Eden's Promise:585D:/
+107.4 "Ferostorm" sync /:Eden's Promise:585E:/
+118.1 "Conflag Strike" sync /:Eden's Promise:585D:/
+128.7 "Ferostorm" sync /:Eden's Promise:585E:/
+140.1 "Maleficium" sync /:Eden's Promise:5872:/
+145.9 "--sync--" sync /:Eden's Promise:585A:/ window 145.9
+148.4 "--untargetable--"
+
+# Intermission. Can be pushed, unknown threshold, maybe 70% HP?
+196.5 "Eternal Oblivion" sync /:Eden's Promise:587A:/ window 50,10
+204.3 "--sync--" sync /:Eden's Promise:4B20:/
+232.9 "Earth Shaker" sync /:Eden's Promise:5885:/
+232.9 "Obliteration" sync /:Eden's Promise:4B43:/
+251.2 "Eternal Oblivion" sync /:Eden's Promise:587B:/ window 20,20
+259.0 "--sync--" sync /:Eden's Promise:4B21:/
+267.3 "Classical Sculpture" sync /:Chiseled Sculpture:5886:/
+290.2 "Rapturous Reach" sync /:Eden's Promise:5887:/
+290.2 "Palm Of Temperance" sync /:Eden's Promise:4B44:/
+308.4 "Eternal Oblivion" sync /:Eden's Promise:587C:/ window 20,20
+316.2 "--sync--" sync /:Eden's Promise:4B22:/
+337.5 "Laser Eye" sync /:Eden's Promise:4B47:/
+356.3 "Eternal Oblivion" sync /:Eden's Promise:587D:/ window 20,20
+364.1 "--sync--" sync /:Eden's Promise:4B23:/
+
+
+# Post-intermission initial block.
+500.0 "--sync--" sync /:4B48:Eden's Promise/ window 500,10
+504.9 "Paradise Lost" sync /:Eden's Promise:4B48:/
+511.9 "--targetable--"
+522.0 "Initialize Recall" sync /:Eden's Promise:5859:/ window 30,30
+534.1 "Stock" sync /:Eden's Promise:5860:/
+542.8 "Formless Judgment" sync /:Eden's Promise:5873:/ window 30,30
+545.6 "--sync--" sync /:Eden's Promise:5879:/
+555.7 "Release" sync /:Eden's Promise:5861:/
+
+
+568.6 "Stock" sync /:Eden's Promise:5860:/ window 15,30
+578.3 "Junction Shiva?" sync /:Eden's Promise:5862:/ jump 700.0
+578.3 "Junction Titan?" sync /:Eden's Promise:5863:/ jump 800.0
+589.0 "Diamond Dust?"
+589.0 "Earthen Fury?"
+594.0 "Impact 1?"
+597.0 "Impact 2?"
+600.2 "Frigid Stone?"
+
+
+# Paths split here. Titan OR Shiva can be used first,
+# and then this first block, always ending in 5861 Release,
+# is followed by:
+
+# Unknown_5879 > Cast > Cast > Malefecium
+# The time between 5861 Release and Unknown 5879
+# is consistent at 9 seconds for both Shiva and Titan.
+
+# It will then continue to the other of Titan/Shiva,
+# which will in turn be followed by the other closing pattern:
```

Comment:
Sorry, I'm not quite sure that I follow this loop here.

So one potential option is:
Titan> 5861 Release > 5879 > Cast > Cast > Maleficium > Shiva > Stock > Formless Judgment > 5879 > Release?

Then ... random Shiva or Titan after, or repeat?

Sorry for this suggestion, but I think this would be clearer to just write out the whole Shiva or Titan branch separately, rather than having mini blocks with only a few abilities in them.

Reply:
Sure. It's like this:
```
1. Post-intermission stuff
2. Shiva OR Titan
3. Release > Cast > Cast >Maleficium
4. Titan OR Shiva
5. Stock > Formless Judgment > 5879 > Release
6. Shiva OR Titan
7. Release > Cast > Cast >Maleficium
etc
```

The issue is that lines 3/5/7 are always encountered in the order seen in this table, but lines 2/4/6 can be switched. Thus, there's no way to "know" whether Shiva or Titan will be followed by Maleficium or Formless Judgment. If you prefer to duplicate the Maleficium/Judgment blocks and append them to both Titan/Shiva, we could do that.

Reply:
Ahhh I see now, so you'd need to duplicate everything and have Titan > Maleficium and Titan > Formless Judgement as well as Shiva > Maleficium and Shiva > Formless Judgment.  This is probably fine, then.